nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/blob/MoveBlobToColdStorage.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *  
+ * Contributors:
+ *     Salem Aouana
+ */
+
+package org.nuxeo.ecm.automation.core.operations.blob;
+
+import static org.nuxeo.ecm.core.blob.DocumentBlobColdStorageHelper.FILE_CONTENT_PROPERTY;
+import static org.nuxeo.ecm.core.blob.DocumentBlobColdStorageHelper.moveBlob;
+
+import org.nuxeo.ecm.automation.core.Constants;
+import org.nuxeo.ecm.automation.core.annotations.Context;
+import org.nuxeo.ecm.automation.core.annotations.Operation;
+import org.nuxeo.ecm.automation.core.annotations.OperationMethod;
+import org.nuxeo.ecm.automation.core.collectors.BlobCollector;
+import org.nuxeo.ecm.core.api.Blob;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.core.api.DocumentModel;
+
+/**
+ * Moves the {@link Blob} associated with the {@link DocumentModel} to the cold storage.
+ * 
+ * @since 11.1
+ */
+@Operation(id = MoveBlobToColdStorage.ID, category = Constants.CAT_BLOB, label = "Move Document Blob To ColdStorage", description = "Moves the blob document content to the cold storage.")
+public class MoveBlobToColdStorage {
+
+    public static final String ID = "Blob.MoveBlobToColdStorage";
+
+    @Context
+    protected CoreSession session;
+
+    @OperationMethod(collector = BlobCollector.class)
+    public Blob run(DocumentModel doc) {
```

Comment:
I would rather return the updated document, like the `UpdateDocument` operation, rather than the main content which is arbitrary.

Reply:
yep, then i will move the `moveContentToColdStorage ` under the document package instead the `blob` as the input/output a document.  Are you agree ?

Reply:
yes